server/broadcast.go

```diff
@@ -33,18 +35,27 @@ func (pi *packetInfo) broadcastMessage() {
 			pi.broadcastAll(msgs)
 		} else {
 			sendingPlayer := pi.tracker.playerByConnection(pi.conn)
+			if sendingPlayer == nil {
+				sendingPlayer = pi.tracker.findPlayerInPoolSnapshots(pi.conn)
+				log.Debugf(logDirectMessage+"Found %v for missing direct message sender\n", sendingPlayer)
```

Comment:
s/missing/disconnected ?

Reply:
Yeah I was conflicted about what to write here. Since we don't ever actively disconnect clients, it's not technically always true, but that's probably not a helpful distinction.

disconnected is easier to understand. I will put that.